drivers/sensor/lsm303dlhc_accel/lsm303dlhc_accel.c

```diff
@@ -83,7 +155,7 @@ static int lsm303dlhc_accel_init(struct device *dev)
 	drv_data->i2c = device_get_binding(config->i2c_name);
 	if (drv_data->i2c == NULL) {
 		LOG_ERR("Could not get pointer to %s device",
-			    config->i2c_name);
+			config->i2c_name);
```

Comment:
Allright, but I should still keep them in this PR?

Reply:
It would have been fine to add a new commit to this PR, but I went ahead and merged since it doesn't depend on the whitespace changes. Feel free to submit another PR.